batch_functions/batch_filter.m

```diff
@@ -26,10 +26,10 @@
             %%           Filter           %%
             %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
             if ~dir_config.use_raw
-                [filtered_map, label_log] = filter_continuous(selected_data, ...
-                    sample_rate, dir_config.notch_filt, dir_config.notch_freq, ...
-                    dir_config.notch_bandwidth, dir_config.filt_type, ...
-                    dir_config.filt_freq, dir_config.filt_order);
+                    [filtered_map, label_log] = filter_continuous(selected_data, ...
+                        sample_rate, dir_config.notch_filt, dir_config.notch_freq, ...
+                        dir_config.notch_bandwidth, dir_config.filt_type, ...
+                        dir_config.filt_freq, dir_config.filt_order);
```

Comment:
Unnecessary change altogether. These changes actually add an additional level of indentation than it should have should be undone.